cluster_management/src/main/java/com/pinterest/rocksplicator/task/BackupTask.java

```diff
@@ -116,7 +116,7 @@ protected void executeBackup(String host, int port, String dbName, String storeP
       throws RuntimeException {
     try {
       if (useS3Store) {
-        Utils.backupDBToS3WithLimit(host, port, dbName, backupLimitMbs, s3Bucket, storePath);
+        Utils.backupDBToS3WithLimit(host, port, dbName, backupLimitMbs, s3Bucket, storePath, shareFilesWithChecksum);
```

Comment:
nit: have u formatted this in Intellij with Pinterest style? this line seem quite long

Reply:
formatted code written per Pinterest style